INSTALL.rst

```diff
@@ -1,123 +1,104 @@
-  cd scikit-image
-  pip install -e .
-
-To update the installation::
-
-  git pull  # Grab latest source
-  pip install -e .  # Reinstall
-
-Platform-specific notes follow below.
-
-a. Windows
-``````````
-If you experience the error ``Error:unable to find vcvarsall.bat`` it means
-that your computer does not have recommended compilers for Python. You can
-either download and install Windows compilers from `here`_  or use
-`MinGW compilers`_ . If using `MinGW`, make sure to correctly configure
-``distutils`` by modifying (or create, if not existing) the configuration file
-``distutils.cfg`` (located for example at
-``C:\Python26\Lib\distutils\distutils.cfg``) to contain::
+Now, whenever you want to use this specific version of scikit-image, make sure
+you activate it by::
 
-  [build]
-   compiler=mingw32
+  # Change to the directory where you created the virtual environment
+  cd YOUR_DIRECTORY
+  # Activate it
+  source skimage/bin/activate
+  # Use python as you wish
 
-A run-through of the compilation process for Windows is included in
-our `setup of Azure Pipelines`_ (a continuous integration service).
 
-.. _miniconda: http://conda.pydata.org/miniconda.html
-.. _python.org: http://python.org/
-.. _setup of Azure Pipelines: https://github.com/scikit-image/scikit-image/blob/master/azure-pipelines.yml
-.. _here: https://wiki.python.org/moin/WindowsCompilers#Microsoft_Visual_C.2B-.2B-_14.0_standalone:_Visual_C.2B-.2B-_Build_Tools_2015_.28x86.2C_x64.2C_ARM.29
-.. _venv: https://docs.python.org/3/library/venv.html
-.. _virtual environments: https://docs.python-guide.org/en/latest/dev/virtualenvs/
-.. _MinGW compilers: http://www.mingw.org/wiki/howto_install_the_mingw_gcc_compiler_suite
+Should you need to have access to the wheels offline, they can be downloaded
+manually from `PyPi <https://pypi.org/project/scikit-image/>`__.
 
-b. Debian and Ubuntu
-````````````````````
-Install all the required dependencies::
+conda
+=====
 
-  sudo apt-get install python3-matplotlib python3-numpy python3-pil python3-scipy python3-tk
+When using conda, you may find the following bash commands useful::
 
-Install suitable compilers::
+  # Create a conda environment named ``skimage``
+  conda create --name skimage
+  # Activate it
+  conda activate skimage
+  conda install scikit-image
 
-  sudo apt-get install build-essential cython3
+Now, whenver you want to use this specific version of scikit-image, make sure
+you activate your conda environment by typing::
 
-Complete the general development installation instructions above.
+  # Activate the specific conda environment
+  conda activate skimage
+  # Use python as you wish
 
-Build Requirements
-------------------
-* `Python >= 3.5 <http://python.org>`__
-* `Numpy >= 1.11 <http://numpy.scipy.org/>`__
-* `Cython >= 0.23.4 <http://www.cython.org/>`__
+While the default Anaconda installation provides scikit-image, you may wish to
+explore `conda-forge <https://conda-forge.org/>`__ for a more updated version.
```

Comment:
What if instead of _for a more updated version_ we had _for an updated version_, or _for a newer version_? :)